extensions/web/lib.deno_web.d.ts

```diff
@@ -205,6 +205,28 @@ declare class TextEncoder {
   readonly [Symbol.toStringTag]: string;
 }
 
+declare class TextDecoderStream {
+  /** Returns encoding's name, lowercased. */
+  readonly encoding: string;
+  /** Returns `true` if error mode is "fatal", and `false` otherwise. */
+  readonly fatal: boolean;
+  /** Returns `true` if ignore BOM flag is set, and `false` otherwise. */
+  readonly ignoreBOM = false;
+  constructor(
+    label?: string,
+    options?: { fatal?: boolean; ignoreBOM?: boolean },
+  );
+  readonly readable: ReadableStream<string>;
+  readonly writable: WritableStream<BufferSource>;
+}
+
+declare class TextEncoderStream {
+  /** Returns "utf-8". */
+  readonly encoding = "utf-8";
```

Comment:
```suggestion
  readonly encoding: "utf-8";
```

Reply:
The definition for `TextEncoder` has `readonly encoding = "utf-8";`, with an equals sign. Should both be changed?

Reply:
Yes, sorry.  Didn't catch that one.